packages/message-system/src/config/config.v1.json

```diff
@@ -264,6 +264,40 @@
                     }
                 }
             }
+        },
+        {
+            "conditions": [
+                {
+                    "os": {
+                        "macos": "!",
+                        "linux": "!",
+                        "windows": "!",
+                        "android": "*",
+                        "ios": "!",
+                        "chromeos": "!"
+                    },
+                    "browser": {
+                        "firefox": "!",
+                        "chrome": ">105",
+                        "chromium": ">105"
+                    }
+                }
+            ],
+            "message": {
+                "id": "b6c6dbeb-dca2-4481-b421-0381d2bd4927",
+                "priority": 91,
+                "dismissible": true,
+                "variant": "warning",
+                "category": "banner",
+                "content": {
+                    "en-GB": "USB connections are currently unavailable for Chrome on Android. Please use the desktop app instead.",
+                    "en": "USB connections are currently unavailable for Chrome on Android. Please use the desktop app instead.",
```

Comment:
what about
```suggestion
                    "en": "USB connections are unavailable for current Chrome on Android. Please use the desktop app instead.",
```
?

Reply:
@sime

Reply:
Yes, and it matched the text we agreed upon in Slack :)